README.md

```diff
@@ -122,6 +122,7 @@ React.run(main(component.flags), dom => render(dom, document.getElementById('app
 
 ## Examples
 
+- [Composability](examples/Composability/index.tsx)
```

Comment:
@IMax153 the folder name is `ComposeModules` not `Composability` and I would move the item at the bottom of the list (it's a more complex example)

Reply:
See https://github.com/gcanti/elm-ts/pull/44/commits/b349c6de357aa1b128b6df251d2923a73d4c1501